deeppavlov/models/kbqa/tree_parser.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from io import StringIO
+from typing import List, Tuple
+
+from udapi.block.read.conllu import Conllu
+from udapi.core.node import Node
+from ufal_udpipe import Model as udModel, Pipeline
```

Comment:
Make a separate requirements file for `ufal_udpipe` and `udapi` and add it to the config:
```"requirements": [
      "{DEEPPAVLOV_PATH}/requirements/tf.txt",
      "{DEEPPAVLOV_PATH}/requirements/fasttext.txt",
      "{DEEPPAVLOV_PATH}/requirements/udpipe.txt"
    ],```